staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/types.go

```diff
@@ -68,6 +68,11 @@ type APIServiceSpec struct {
 	// The primary sort is based on VersionPriority, ordered highest to lowest (20 before 10).
 	// The secondary sort is based on the alphabetical comparison of the name of the object.  (v1.bar before v1.foo)
 	// Since it's inside of a group, the number can be small, probably in the 10s.
+	// In case of equal version priorities, the version string will be used to compute the order inside a group.
+	// If the version string is kube-like, it will be used for ordering otherwise lexical order will be used.
```

Comment:
If the version string is "kube-like", it will sort above non "kube-like" version strings, which are ordered lexicographically. "Kube-like" versions start with a "v", then are followed by a number (the major version), then optionally the string "alpha" or "beta" and another number (the minor version). These are sorted first by GA > beta > alpha, and then within each group by comparing major version, then minor version. An example sorted list of versions: v10, v2, v1, v11beta2, v10beta3, v3beta1, v12alpha1, v11alpha2, foo1, foo10.